auto/sources

```diff
@@ -19,6 +19,7 @@ NJS_LIB_SRCS=" \
    src/njs_utils.c \
    src/njs_chb.c \
    src/njs_value.c \
+   src/njs_cnst.c \
```

Comment:
Style: we do not use abbreviations for short names, especially if it gives us just one letter saving.
Style: I propose to name it as it was originally named `njs_predefined`. The renaming includes finename, typedef, variable names.